apps/etterlatte-brreg/src/main/kotlin/no/nav/etterlatte/enhetsregister/EnhetsregisterRoute.kt

```diff
@@ -33,4 +33,9 @@ fun Route.enhetsregApi(service: EnhetsregService) {
             call.respond(service.hentStatsforvalterListe())
         }
     }
-}
\ No newline at end of file
+}
+
+internal fun vaskInput(input: String?, orElse: () -> String) = requireNotNull(input) {
```

Comment:
Dette løser vel problemet med det Snyk klager på, men hadde det vært ryddigere og lagt til enkel validering av input bare? `orgnr` er egentlig bare tall (og evt. spaces), og `enheter` trenger bare bokstaver + spaces som tillatt?

Reply:
Verksemder kan heite alt mogleg rart, med diverse spesialteikn i namnet 😞 

Eit alternativ er jo sjølvsagt å gjera denne mindre generisk. Ser ut som vi i dag kun bruker han for å hente ut alle statsforvaltarar 🤔